projects/addon-doc/components/main/main.style.less

```diff
@@ -91,11 +92,31 @@ tui-doc-navigation.tui-doc-navigation {
     }
 }
 
+.tui-doc-aside {
+    position: fixed;
+    top: 6rem;
+    right: 1rem;
+    width: @aside-width;
+
+    @media @tui-tablet {
```

Comment:
I feel like it still takes up a lot of space on smaller desktop resolutions, maybe let's only show it on wider screens?